### ISSUE TITLE
clarification of language-git

### ISSUE THREAD

**Issue by garretwilson** [PrismJS/prism#862]
This is a request for clarification.

The [Prism documentation](http://prismjs.com/) indicates that Prism supports `language-git`. But what language is that? Is this that Git commands in the console? Is that Git files such as `.gitignore`?

**Comment by jordandanielewski** [comment]
You can find more infos here https://github.com/PrismJS/prism/blob/gh-pages/components/prism-git.js

**Comment by garretwilson** [comment]
Oh, it's _output_ from the Git command. That's sort of interesting.

It should would be helpful it something was mentioned on the main page, or if there was a link to the prism-git file, or something.

Cheers!

**Comment by LeaVerou** [comment]
Yes, we need a page with doc info for all languages, not just this. Perhaps we should add a description field to `components.js` and have a page that lists all languages and their descriptions. Currently the only listing is the download page. This design worked well 3 years ago when Prism had a few languages, but is very confusing now...
